RecoTauTag/HLTProducers/python/applyL2TauTag.py

```diff
@@ -0,0 +1,82 @@
+import FWCore.ParameterSet.Config as cms
+from HLTrigger.Configuration.customizeHLTforPatatrack import customizeHLTforPatatrackTriplets
+from RecoTauTag.HLTProducers.l2TauNNProducer_cfi import *
+from RecoTauTag.HLTProducers.l2TauTagFilter_cfi import *
+
+def update(process):
+    thWp = {
+            'WP_Tight': 0.180858813224404,
+            'WP_Medium': 0.12267940863785043,
+            'WP_Loose': 0.08411243185219064,
+    }
+
+    working_point = "Tight"
+    rateWP=("WP_{}").format(working_point)
+    graphPath = 'RecoTauTag/TrainingFiles/L2TauNNTag/L2TauTag_Run3v1.pb'
+
+    normalizationDict = 'RecoTauTag/TrainingFiles/L2TauNNTag/NormalizationDict.json'
+
+
+    process = customizeHLTforPatatrackTriplets(process)
```

Comment:
```python
if 'statusOnGPU' not in process. __dict__:
    process = customizeHLTforPatatrackTriplets(process)
```

we will need to check how the performance change if we use only quadruplets (i.e. calling `customizeHLTforPatatrack` instead)